src/protocol/thread/pause.ts

```diff
@@ -451,9 +451,18 @@ export class Pause {
       return this.loadMouseTargetsWaiter.promise;
     }
     this.loadMouseTargetsWaiter = defer();
-    const { elements } = await this.sendMessage(client.DOM.getAllBoundingClientRects, {});
-    this.mouseTargets = elements;
-    this.loadMouseTargetsWaiter.resolve();
+    let rv;
+    try {
+      rv = await this.sendMessage(client.DOM.getAllBoundingClientRects, {});
+      this.mouseTargets = rv.elements;
+    } catch (e) {
+      console.error("DOM.getAllBoundingClientRects failed", e);
+    } finally {
+      this.mouseTargets = [];
+      this.loadMouseTargetsWaiter.resolve();
```

Comment:
This needs to resolve() with the return value because that is what will be returned when this.loadMouseTargetsWaiter is set on entry.  This also doesn't need to belong in the finally clause because the catch block ensures we won't be throwing here.